src/Dialogs/ContentWarningDialog.vala

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2018 elementary, Inc. (https://elementary.io)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+namespace AppCenter.Widgets {
```

Comment:
You don't need this. This is for when you want to have multiple classes in a single file. Instead you can do `public class AppCenter.Widgets.ContentWarningDialog : Gtk.Dialog {`